app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalServerDecorator < MiqDecorator
+  def fileicon
+    "svg/vendor-#{vendor.try(:downcase)}.svg"
```

Comment:
I think if that is happening, it's a backend bug that needs to be fixed.

A PhysicalServer without a vendor does not quite make sense... (and makes even less sense now, when we only have one possible vendor :)).